homeworks/01-bubble-sort/src/utils/appUtils.test.ts

```diff
@@ -0,0 +1,19 @@
+import { createRandomNumberArray } from './appUtils';
+
+describe('appUtils', () => {
```

Comment:
Nice to see tests 👍

But they would be more applicable to `sort` functionality.

Reply:
Yes, i will open an issue to implement unit tests for sort functionality. Will implement later.

Reply:
https://github.com/vbarbasins/evo-ts-bootcamp/issues/5